src/main/ui/app/index.jsx

```diff
@@ -13,8 +13,15 @@ import {render} from "react-dom";
 import {Router, hashHistory} from "react-router";
 import routes from "./routes";
 import "./sass/addonindex.scss";
+import ReactGA from 'react-ga';
+
+ReactGA.initialize('UA-16695719-3');
+
+function trackView() {
+    ReactGA.pageview(window.location.pathname + window.location.search);
```

Comment:
We should only log these page views for the prod server (i.e. if it's not configured, then don't log anything)

Reply:
How can we know on which environment are we running the app?

Reply:
You don't actually need to know that. Basically, you only want to log things if a (to be created) config setting like googleanalytics.url is set

Reply:
In which file should I place this property?

Reply:
@BartlomiejRasztabiga I can see that you haven't defined the variable in the right place i.e. the "external" key within which you defined the variable is to be used only for listing the external libraries that would be required as mentioned in the comment there. You need to define this variable below this https://github.com/openmrs/openmrs-contrib-addonindex/blob/02f924dd370071b4e65edf8595bae4d4af2e33bf/src/main/ui/webpack.config.js#L12